src/js/view/controls/tizen/tizen-controls.ts

```diff
@@ -38,21 +43,32 @@ class TizenControls extends Controls {
 
         this.context = context;
         this.playerContainer = playerContainer;
+        this.api = null;
+        this.model = null;
         this.div = null;
         this.backdrop = null;
         this.pauseDisplay = null;
         this.displayContainer = null;
         this.controlbar = null;
+        this.seekbar = null;
+        this.seekState = false;
         this.settingsMenu = null;
         this.showing = false;
         this.instreamState = false;
         this.keydownCallback = null;
+        this.handleKeydown = this.handleKeydown.bind(this);
```

Comment:
I don't think it's good to rewrite (or shadow prototype) properties like this. Either make this bound version `this.keydownCallback`:
```suggestion
        this.keydownCallback = this.handleKeydown.bind(this);
```
or do it below instead of aliasing handleKeydown with `this.keydownCallback = this.handleKeydown` (no purpose in having two properties with the exact same value).

Reply:
Great call!